modules/integration_azure_firewall/conf/00-heartbeat.yaml

```diff
@@ -0,0 +1,13 @@
+## Example
+module: firewall
+name: heartbeat
+
+transformation: false
```

Comment:
except if you really want to make this impossible to customize I would set it to true to provide a variable to user to override default behavior